app/src/conferences/templates/conferences/user_preferences_index.html

```diff
@@ -114,7 +118,7 @@ <h5>{% trans "Bonus" %}</h5>
             },
             success: function(data) {
                 paymentInfo[id].bonus = data['bonus']
-                $('#bonus-value-' + id).text('Bonus: ' + data['bonus']);
+                $('#bonus-value-' + id).text('Bonus: ' + data['bonus'] + ' min');
```

Comment:
I don't like this solution. We have `Bonus: XX min` in two different places (here and in line 49). I suggest to change it in such a way that here you will change only numerical value.